modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/OperationModelParser.java

```diff
@@ -111,6 +111,13 @@ public interface OperationModelParser extends SemanticTermsParser, AdditionalPro
    */
   boolean isConnected();
 
+  /**
+   * @return whether this operation requires a connection provided to it from an associated connection provider
```

Comment:
rephrase: `whether this operation requires connections obtained through a {@link ConnectionProvider} defined in this same extension`

Reply:
done